src/main/resources/application-local.yaml

```diff
@@ -93,7 +93,10 @@ hapi:
     ### the deepest folder level will be used. E.g. - if you put file:/foo/bar/bazz as value then the files are resolved under /static/bazz/**
     #staticLocation: file:/foo/bar/bazz
     ### enable to set the Server URL
-    server_address: https://localhost:8080/oauth/fhir
+    # ******* IMPORTANT *******
+    # Adjusting server_address requires updating the proxy string replace function in the API Project
```

Comment:
It'd be good to add at least the filename so its easier to find it: `fhir-r4-proxy.ts`

or relative path within OSS: `packages/api/src/routes/medical/fhir-r4-proxy.ts`

Reply:
Done